doc/source/visualization.rst

```diff
@@ -622,6 +645,10 @@ See the :meth:`scatter <matplotlib.axes.Axes.scatter>` method and the
 Hexagonal Bin Plot
 ~~~~~~~~~~~~~~~~~~
 
+.. versionadded:: 0.17.0
+
+You can create hexagonal bin plots with :meth:`DataFrame.plot.hexbin`.
+
 .. versionadded:: 0.14
 
 You can create hexagonal bin plots with :meth:`DataFrame.plot` and
```

Comment:
Yes I did a duplicate of the descriptions for `versionadded`. I'm open to suggestions to rewrite in a simpler manner.